aws/index.ts

```diff
@@ -88,7 +77,7 @@ export const kafkaBootstrapBrokers = kafkaCluster?.bootstrapBrokers;
 export const kafkaBootstrapServerSecret = bootstrapServersSecretName;
 
 let jdbcSecretName: string | null = null;
-let jdbc: model.JdbcDatabase | null = null;
+let jdbc: eks.AuroraRdsDatabase | null = null;
```

Comment:
Idem JDBC vs Aurora.

Should this variable support a user-brought database other than Aurora (maybe even RDS)?

Reply:
I guess we don't need these Pulumi templates to support all possible combinations, just one to get people started. 

LGTM

Reply:
> I guess we don't need these Pulumi templates to support all possible combinations, just one to get people started.

This! The template is more like a starting point, but users need some level of customization, then need to change it instead of just change some configuration/parameter.